orc/src/main/java/org/apache/iceberg/orc/ORCSchemaUtil.java

```diff
@@ -214,6 +214,7 @@ public static Schema convert(TypeDescription orcSchema) {
         "Error in ORC file, children fields and names do not match.");
 
     List<Types.NestedField> icebergFields = Lists.newArrayListWithExpectedSize(children.size());
+    // TODO how we get field ids from ORC schema
```

Comment:
I just noticed the logic here and it's a correctness bug. ORC should not assign column IDs when one is missing. Instead, it should ignore the field.

Reply:
Should we use another PR to fix this?